services/graph/pkg/service/v0/users.go

```diff
@@ -1014,3 +986,34 @@ func isValidUserType(userType string) bool {
 
 	return false
 }
+
+func (g Graph) searchOCMAcceptedUsers(ctx context.Context, odataReq *godata.GoDataRequest) ([]*libregraph.User, error) {
+	users := []*libregraph.User{}
+	if g.config.IncludeOCMSharees {
```

Comment:
Not critical, but you should we reverse logic here:
```golang
if !g.config.IncludeOCMSharees {
    return users, nil
}
```
That way you don't have the `if` clause spanning the whole function and the effect of the `if` clause is obvious immediately. But as stated: Not critical if you like it more this way.